.gitignore

```diff
@@ -1,4 +1,5 @@
 .bundle/
+Gemfile.lock
```

Comment:
It should not be versionned in git

Reply:
OK. will remove it.

Reply:
Yes, therefore I added the Gemfile.lock to the .gitignore so that it will not be put to git.